lib/cdo/script_constants.rb

```diff
@@ -114,6 +114,7 @@ module ScriptConstants
       CSD4_2018_NAME = 'csd4-2018'.freeze,
       CSD5_2018_NAME = 'csd5-2018'.freeze,
       CSD6_2018_NAME = 'csd6-2018'.freeze,
+      CSD_POST_SURVEY_2018_NAME = 'csd-post-survey-2018'.freeze,
```

Comment:
without this line, the survey shows up in the "other" section rather than the CSD section of the course selector dropdown in the Progress and Assessment tabs.

we should really be pulling this info from the database and caching it, rather than having script constants defined.